### PR TITLE
[New game]: Bingo

### LINKED ISSUE #4313: [New game]: Bingo
### 🎮 Game Request

Game logic and basic description: Bingo is a social game where players are given cards with grids of numbers. The main goal is to mark off a specific pattern of numbers on the card, usually five in a row, column, or diagonal before your opponents do. The player who completes the pattern and shouts “Bingo!” is declared the winner.

### Point down the features

Game points: Grid Layout: 5x5 grid with B-I-N-G-O columns.
Number Ranges: B (1-15), I (16-30), N (31-45), G (46-60), O (61-75).
Number Drawing: Randomly selects and announces numbers/letters.
Verification: Checks winning cards.
Rewards: Non-monetary prizes for winners.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#4313

## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes #4313

<hr>
 
## Mark the task you have completed ✅

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [X] I have added screenshot for website preview in assets/images 
- [X] I have added entries for my game in main README.md
- [X] I have added README.md in my folder 
- [X] I have added working video of the game in README.md (optional)
- [X] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

image

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Anjaliavv51 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Anjaliavv51,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Comment by Anjaliavv51** [comment]
@ishita-43 @kunjgit please review the Game

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Anjaliavv51,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Comment by Anjaliavv51** [comment]
@ishita-43 @kunjgit please review the Game

**Comment by Anjaliavv51** [comment]
@ishita-43 @kunjgit please review the Game

**Comment by Anjaliavv51** [comment]
@ishita-43 @kunjgit please review the Game

**Comment by Anjaliavv51** [comment]
@kunjgit @ishita-43 please review the Game

File: assets/images/image.png
Comment (ishita-43) [inline assets/images/image.png:1]: rename this image name as your game name read contribution .md for naming convention
I hope you will make changes soon!!

File: README.md [hunk 1]
```diff
@@ -342,6 +340,7 @@ This repository also provides one such platforms where contributers come over an
 | [Taash_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Taash_Game) |
 | [Number_Guessing_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Number_Guessing_Game) |
 
+
```
Comment (ishita-43) [inline README.md:343]: add your game name here and see above entries for reference you need to follow the same format!!

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by Anjaliavv51** [comment]
@ishita-43 review the changes

File: assets/images/image.png
Comment (ishita-43) [inline assets/images/image.png:1]: rename as the name of your game folder

**Comment by kunjgit** [comment]
Thank you  @Anjaliavv51 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**